packages/devextreme/js/events/core/events_engine.js

```diff
@@ -113,7 +113,11 @@ const eventsEngine = injector({
         if(!noBubble) {
             const parents = [];
             const getParents = function(element) {
-                const parent = element.parentNode ?? element.host;
+                const parent = element.parentNode
+                    ?? (typeof element.host === 'object'
```

Comment:
We can use our utils for this: https://github.com/DevExpress/DevExtreme/blob/23_2/packages/devextreme/js/core/utils/type.js#L47
Also, I think we can't use different logic for same issue: https://github.com/DevExpress/DevExtreme/blob/23_2/packages/devextreme/js/core/utils/dom.js#L17

Reply:
isObject() - is good